applications/tari_app_grpc/src/generated/tari.rpc.rs

```diff
@@ -683,6 +697,20 @@ pub mod base_node_client {
             let path = http::uri::PathAndQuery::from_static("/tari.rpc.BaseNode/GetTipInfo");
             self.inner.unary(request.into_request(), path, codec).await
         }
+
+        #[doc = " Search for blocks containing the specified kernels"]
```

Comment:
`///` is the same as `doc` attr 
```suggestion
        /// Search for blocks containing the specified kernels
```

Same thing in a number of other places

Reply:
ah that was generated, i'll check to make sure it's consistent with the rest

Reply:
All the generated implementations in the file end up with the doc attr https://github.com/tari-project/tari/blob/86b4e66b262de07c9f54f6372f7e2240974c0b96/applications/tari_app_grpc/src/generated/tari.rpc.rs#L519